### DISCUSSION TITLE
Use services view instead of dedicated k8s view

### DISCUSSION THREAD

**Discussion by gayanper** [redhat-developer/intellij-kubernetes#490] [Ideas]
In IntelliJ similar integration are done into Services view. For example Docker in IJ CE. Is there any reason the k8s plugin is not providing the content and navigation inside the service view ?

**Comment by adietish** [comment]
Hi Gayanper

Thanks for the input, appreciate it!
Sorry for the late answer. We didnt see your discussion until lately :(

We decided against the service view because we want to match the experience in vscode-kubernetes-tools for a seamless transitition between vscode and IDEA. A toolkit window also is to us the better match because there's more available vertical space to display all the items that are available in your kubeconfig. Imho this is even more important if there's a decision to allow several active contexts at the same time (several users seem to request this).
I dont see the benefit of being restricted to the Services view beside being "standard" compliant. Do I miss something?

**Reply by gayanper** [reply]
No i agree with you. Switch to vscode as my main development tool, i feel that the vscode UX provides much fluent UX experience than IJ currently

**Reply by adietish** [reply]
ok, nice that having a similar UI is of any use :)
What portion feels more fluent to you, where should we improve? Serious question, all we want is to match what users expect, what helps them. So whatever you can report, we're very grateful.
